scene/2d/tile_map.cpp

```diff
@@ -1207,6 +1207,27 @@ Array TileMap::get_used_cells() const {
 	return a;
 }
 
+Array TileMap::get_used_cells_by_id(int id) const {
```

Comment:
In Godot we tend to prefer function parameters with `p_`, so it would be good if you could name this one `p_id`.